flight/flight-sql-jdbc-core/pom.xml

```diff
@@ -147,6 +147,12 @@ under the License.
       <groupId>org.checkerframework</groupId>
       <artifactId>checker-qual</artifactId>
     </dependency>
+
+    <dependency>
+      <groupId>com.github.ben-manes.caffeine</groupId>
+      <artifactId>caffeine</artifactId>
```

Comment:
Do we really need caffeine here ? I don't see obvious usage in other part of this PR.

Reply:
It is used in FlightClientCache.java.

Reply:
Oh yeah, I see it now, sorry for the noise 😄